plugins/systemair/__init__.py

```diff
@@ -53,6 +55,7 @@ def init_serial_connection(self, serialport, slave_address):
             return False
 
     def _read_modbus(self):
+        self._lockmb.acquire()
```

Comment:
In case pf exception it is called, yes. But the code looks like the method just can return false and the there is no release. If the method always throws an exception the if is not required.

Is there any reason why you check the result of the `init_serial_connection()`?